crates/red_knot_python_semantic/resources/mdtest/assignment/unbound.md

```diff
@@ -14,11 +14,14 @@ Name lookups within a class scope fall back to globals, but lookups of class att
 
 ```py
 x = 1
+
```

Comment:
I find the extra vertical spacing more of a negative than a positive in these tests, but it's not a big negative, and I'm OK with it for the sake of consistent formatting.

Reply:
Yeah, same. It would be nice if we could get blacken-docs to think of these as `pyi` stub files, but not sure if that's possible. I find having an autoformatter for local use and CI more important than the specific formatting decisions it makes 😆

Reply:
Black does have an option for this (`--pyi`); not sure if it carries through to blacken-docs.

Reply:
Unfortunately it seems like that's not one of the flags supported by blacken-docs, and it seems like it ignores any black configuration files (https://github.com/astral-sh/ruff/pull/13809#discussion_r1806820453)